pubspec.yaml

```diff
@@ -1,12 +1,16 @@
 name: built_html
-version: 0.1.0+1
+version: 0.2.0
```

Comment:
nit: 0.1.1 (this isn't a breaking change)

Reply:
According to https://semver.org you should increase the second number "when you add functionality in a backwards-compatible manner", and you should increase the third number "when you make backwards-compatible bug fixes". What pattern do you use here then? :-)

Reply:
The difference here is this is "pre-release" software (the first number is a zero) - in that case pubs version of semver actually shifts the meaning of all the numbers over by one ;).

A bit confusing yes, and I think not technically consistent with general semver, but supported by pub. From https://www.dartlang.org/tools/pub/versioning:

```
Although semantic versioning doesn’t promise any compatibility between versions prior to 1.0.0, the
Dart community convention is to treat those versions semantically as well. The interpretation of each
number is just shifted down one slot: going from 0.1.2 to 0.2.0 indicates a breaking change, going to
0.1.3 indicates a new feature, and going to 0.1.2+1 indicates a change that doesn’t affect the public
API.
```

Reply:
Thanks. Didn't know that. :-)